raiden/network/transport/udp/udp_transport.py

```diff
@@ -296,15 +204,15 @@ def start_health_check(self, recipient):
                 {'nonce': 0},  # HACK: Allows the task to mutate the object
```

Comment:
we don't need this hack anymore, since the `healthcheck` is part of the transport, could you please remove the `ping_nonce` argument from the `healthcheck` class and just access this dictionary directly?

Reply:
Actually, the only argument the `healthcheck` needs in the target address, everything else is available in the instance state.